src/rard/templates/research/partials/linked_antiquarian.html

```diff
@@ -22,17 +28,17 @@
     {% if can_edit and has_object_lock %}
 
             {% if linked_field == 'fragment' %}
-            <form novalidate class='form-inline' action='{% url "fragment:remove_fragment_link" link.pk %}' method='POST'>
+            <form novalidate class='form-inline' action='{% url "fragment:remove_fragment_link" antiquarian.pk %}' method='POST'>
             {% else %}
-            <form novalidate class='form-inline' action='{% url "testimonium:remove_testimonium_link" link.pk %}' method='POST'>
+            <form novalidate class='form-inline' action='{% url "testimonium:remove_testimonium_link" antiquarian.pk %}' method='POST'>
             {% endif %}
                 {% csrf_token %}
+                 <input type="hidden" name="antiquarian_request" value={{object.pk}} />
                 <div class='form-group'>
                     <button type='submit' class='btn btn-link text-danger confirm-delete p-0 ml-2'
-                        data-what='link'>{% trans 'Remove Link' %}</button>
+                        data-what='antiquarian_link'><small>{% trans 'remove all links' %}</small></button>
```

Comment:
I think this button needs to come with a warning: "Are you sure you want to delete all links associated with this antiquarian and their works". In the same way you get a warning for Make Anonymous, or Make Unlinked

Reply:
oki - does it come up with any alert at all? It has the confirm-delete class so it _should_ but I also noticed it didn't work for me and then it started working for me without me changing the code... 
That might be something to look into if you're not seeing it pop up